build.sbt

```diff
@@ -24,6 +24,16 @@ lazy val compilerSettings = CompilerSettings.options ++ Seq(
 
 lazy val commonSettings = projectSettings ++ coverageSettings ++ compilerSettings
 
+lazy val casper = (project in file("casper"))
+  .settings(commonSettings: _*)
+  .settings(
+    libraryDependencies ++= commonDependencies ++ protobufDependencies,
+    PB.targets in Compile := Seq(
+      scalapb.gen() -> (sourceManaged in Compile).value
+    )
+  )
+  .dependsOn(models, rspace, comm)
+
```

Comment:
I can just do this with my next push but

`.aggregate(casper, crypto, comm, models, regex, rspace, node, rholang, rholangCLI, roscala)`

at the bottom